src/views/Records/Record.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    class="standard container-fluid"
+  >
+    <div v-if="error"
+      class="error"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 9 spaces but found 6 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4961435)